tf2_sensor_msgs/package.xml

```diff
@@ -23,12 +23,11 @@
   <depend>sensor_msgs</depend>
   <depend>tf2</depend>
   <depend>tf2_ros</depend>
-  <!-- <depend>python_orocos_kdl</depend> -->
-  <!-- <test_depend>rostest</test_depend> -->
```

Comment:
I still think we should remove this. It's a ros1 dependency

Reply:
@gleichdick I will relaunch CI when this line will be removed

Reply:
I removed them, furhtermore some more flake8 issues are resolved which the CI found. Maybe we should add the flake8 modules mentioned [here](https://docs.ros.org/en/crystal/Installation/Linux-Development-Setup.html#install-development-tools-and-ros-tools) to the CI of this project (not the big one)?

Reply:
While I agree we should eventually do that, I don't think we should do that here.  The next step for this package in my opinion is to split it up into separate `tf2_sensor_msgs` and `tf2_sensor_msgs_py` packages, and actually get the Python stuff working (which it currently isn't).  At that point, it would make sense to enable the automated flake8 tests.